### ISSUE TITLE
Replay controls widget (gui_replay_controls.lua) misbehaves on ultrawide monitors

### ISSUE THREAD

**Issue by ThingsWillCalmUp** [ZeroK-RTS/Zero-K#5394]
RE: This discussion on discord: https://discord.com/channels/278805140708786177/432874986064052235/1316560761623023638

TL;DR:
The replay controls set the window Y offset based on a value calculated by window width, which causes them to hide beneath the minimap or other elements when ZK is running at ultrawide resolutions.

Someone should probably fix this properly and make the widget able to save its location after being dragged around. For now, I propose the following changes:

gui_replay_controls.lua::L119-120:
```
	local screenWidth, screenHeight = Spring.GetViewGeometry()
	local windowY = math.floor(screenHeight/8)
``` 

This will probably place the widget over the user's playerlist, but it should be draggable from there. Worst case as a dirty hack is to change the window's "right" value to something higher as well, slap it in the middle of the display, and rely on the user to relocate it every time, because it'd at least be somewhere grabbable at that point.

**Comment by GoogleFrog** [comment]
Sounds like something to fix. Saving the location is a bit worrying since it might end up saved somewhere inaccessible. We'd probably want a type of window that doesn't save if it is too far behind another window.

**Comment by ThingsWillCalmUp** [comment]
Okay, I put in a PR to do this. I THINK I did it right? Please let me know if I need to fix/change anything.

https://github.com/ZeroK-RTS/Zero-K/pull/5400
